NasaFacilities.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
```

Comment:
We should move these to .gitignore

Reply:
do you think so? it's not connected to user